contrib/docs/tutorials/aws-alb-aad-ocid-auth.md

```diff
@@ -41,13 +41,14 @@ Use the following advanced settings:
 - `Scope` = `openid profile offline_access`
 - `Action on unauthenticated request` = `Autenticate (client reattempt)`
 
-Once you've saved the action you should see an authentication flow be triggered against AAD when visiting Backstage address at `https://backstage.yourdomain.com`, the flow will not complete successfully as Backstage app isn't yet configured properly.
+Once you've saved the action, you should see an authentication flow be triggered against AAD when visiting Backstage address at `https://backstage.yourdomain.com`. The flow will not complete successfully as Backstage app isn't yet configured properly.
 
 ## Backstage changes
 
 ### Frontend
 
-The Backstage App requires a SingInPage when authentication is required. When using ALB authentication Backstage will only be loaded once the user has successfully authenticated; we won't need to display a SignIn page, however we will need to create a dummy SignIn component that can refresh the token.
+The Backstage App requires a SingInPage when authentication is required. 
```

Comment:
using 'requires' and 'required'... it feels like we can use a better wording here